### ISSUE TITLE
Volta install blocked on Windows 10

### ISSUE THREAD

**Issue by bbkane** [volta-cli/volta#715]
Hello!

I'm trying to install Volta on Windows 10. Per [the website](https://docs.volta.sh/guide/getting-started), I enabled Developer Mode. I even restarted my Surface after just to be sure.

![image](https://user-images.githubusercontent.com/6081085/79679864-6bc40800-81be-11ea-9f5a-dea470a7c73f.png)

But when I download the file, I get the following message:

![image](https://user-images.githubusercontent.com/6081085/79679940-2b18be80-81bf-11ea-946b-31024508022e.png)

Clicking Keep opens the following popup:

![image](https://user-images.githubusercontent.com/6081085/79679971-6c10d300-81bf-11ea-989d-4c5612e4503f.png)

Is there a way to make this app more trustworthy? Is it signed? Can it be published to the Microsoft App Store? Are there easy workarounds (maybe run as Admin?)?

This may be related to #407

Thanks!
Ben

**Comment by charlespierce** [comment]
Hi Ben! Thanks for reporting, and for the screenshots of the warnings that Windows puts up. You’re right that this is a result of #407, so closing in favor of that, but I’ll post an update there as well.

**Comment by bbkane** [comment]
Would you accept a PR updating https://docs.volta.sh/guide/getting-started to mention that the Windows installer must be run with Admin privileges to avoid this?

**Comment by charlespierce** [comment]
@bbkane Does running the installer as an admin completely remove the warnings? If so, I'd be happy to have a PR indicating that. I know the installer will ask for admin permissions when it needs them, but if running it that way from the start can get around the "You shouldn't run this" warnings, that would be great.

**Comment by bbkane** [comment]
@charlespierce I'll check sometime this week on my wife's surface and get back to you (I just married, lots of tasks to complete for a while). From what I remember, Windows won't even let you run it in the first place without it being signed or with admin privileges.
